application/views/login_form.php

```diff
@@ -1,15 +1,20 @@
+<script type="text/javascript">
+    var controller = "<?=site_url('user/login')?>";
+    var registration = "<?=site_url('registration')?>"
+</script>
+
 <div class="signcontainer">
     <h3>高校用户登录</h3>
     <hr/>
     <div class="col-sm-4">用户名</div>
     <div class="col-sm-8">
-        <input type="text" class="form-control" name="mail">
+        <input type="text" class="form-control" name="mail" id="mail">
```

Comment:
I just wanted to use the jQuery selector more conveniently. I didn't know how to select `name` then so I wrote the code like this. I can refer to the docs and improve it if it really matters.